.github/workflows/basic_checks.yaml

```diff
@@ -44,14 +44,14 @@ jobs:
         run: rcmdcheck::rcmdcheck(args = c("--no-manual"), error_on = "warning", check_dir = "check")
         shell: Rscript {0}
         
-      - uses: docker/build-push-action@v1
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: stemangiola/biocasia2020_tidytranscriptomics
-          tag_with_ref: true
-          tag_with_sha: true
-          tags: biocasia2020
+      # - uses: docker/build-push-action@v1
```

Comment:
Do you want me to create the secrets? How come we don't have them this round around?